Source/Scene/Material.js

```diff
@@ -878,13 +888,12 @@ function createTexture2DUpdateFunction(uniformId) {
           : Resource.createIfNeeded(uniformValue);
 
         var promise;
-        if (ktxRegex.test(resource.url)) {
-          promise = loadKTX(resource);
-        } else if (crnRegex.test(resource.url)) {
-          promise = loadCRN(resource);
+        if (ktx2Regex.test(resource.url)) {
```

Comment:
I guess this was already like this, but it means that we require KTX files to have a specific extension, which is generally bad practices (especially on the web) normally we would check the header magic or similar.

Reply:
@lilleyse We discussed this offline - should I open an issue for this?

Reply:
Yeah open an issue

Reply:
opened https://github.com/CesiumGS/cesium/issues/9640